### DISCUSSION TITLE
Are admin routes open in latest versions of Laravel?

### DISCUSSION THREAD

**Discussion by AlexKomMsk** [Laravel-Backpack/community-forum#243] [Q&A (Help)]
Hello. I'm setting up a backpack admin in Laravel 9 with Inertia. When viewing the page code (in chrome browser), I see this:
`const Ziggy = {"url":"http:\/\/sssss.loc","port":null,"defaults":{},"routes":{"backpack.dashboard":{"uri":"crud\/dashboard","methods":["GET","HEAD"]},"backpack":{"uri":"crud","methods":["GET","HEAD"]}`
I would not want to show users this information. Can I somehow hide this information from the browser (hide backpack routes)?

**Comment by pxpm** [comment]
Hello @AlexKomMsk 

I am not a user of Ziggy package myself, but from a quick google search I found out that you can configure the "hidden" urls. https://github.com/tighten/ziggy#filtering-routes

Cheers

**Reply by AlexKomMsk** [reply]
> Hello @AlexKomMsk
> 
> I am not a user of Ziggy package myself, but from a quick google search I found out that you can configure the "hidden" urls. https://github.com/tighten/ziggy#filtering-routes
> 
> Cheers

Hello! The problem is that the backpack doesn't have route names. And Ziggy is filtered only through names.
